exercises/armstrong-numbers/lib/example.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:math' show pow;
+
+class ArmstrongNumbers {
+  bool isArmstrongNumber(final int number) {
+    final String numberAsString = number.toString();
+    final int numOfDigits = numberAsString.length;
```

Comment:
This could've easily been a const , add comment reasoning choice of final

Reply:
Add early exists for `len = 1; return true` and `len = 2; return false` because novice students need to learn early exits are a good idea and help on optimizations too.

Reply:
I disagree about early exists. I prefer one entry, one exit. We are not here to teach people the language, but provide them with a problem and then resources to utilize in their search for a solution.

The example will not be a primary resource for them. Unless they are already familiar with github, they may not consider looking at the track's own solution to a problem.

No where in our documentation do we suggest or promise that the solutions given for an exercise meet a criteria. If you want to have that discussion, then we should open an issue to discuss it.

But I am not going to do more work than we've outlined in our documentation.

Reply:
>No where in our documentation do we suggest or promise that the solutions given for an exercise meet a criteria.

Yes, but I was under the impression that we're supposed introduce better ways to do things in an effort to make students better at dart.

Reply:
No, we do not introduce "better ways" of doing things. The user is never required to follow someone else's footsteps in how a problem is resolved. They would never see the example solution provided unless they seek out the GitHub repo.

We may suggest Dart concepts to use in the exercise's README, but not in the example solution.

Reply:
> This could've easily been a const , add comment reasoning choice of final

So far, I don't see it as being easy. But I'll leave a comment otherwise.